invenio_communities/communities/services/config.py

```diff
@@ -40,6 +38,15 @@ class SearchOptions(SearchOptionsBase):
     }
 
 
+class CommunityMembersLink(RecordLink):
+    """Link variables setter for Community Members links."""
+
+    @staticmethod
```

Comment:
I saw this `staticmethod` defined in `invenio-records-resources`, but it looks weird how it is used! :)